app/uk/gov/hmrc/merchandiseinbaggagefrontend/controllers/ReviewGoodsController.scala

```diff
@@ -36,21 +36,21 @@ class ReviewGoodsController @Inject()(override val controllerComponents: Message
 
   val onPageLoad: Action[AnyContent] = actionProvider.journeyAction { implicit request =>
     request.declarationJourney.goodsEntries.map(Goods.apply) match {
```

Comment:
I think this will blow up if the user has started a journey but jumps ahead to this page with an incomplete goods entry.

In this case it should probably go to /invalid-request